plugins/net_plugin/net_plugin.cpp

```diff
@@ -638,10 +645,10 @@ namespace eosio {
       // @param callback must not callback into queued_buffer
       bool add_write_queue( const std::shared_ptr<vector<char>>& buff,
                             std::function<void( boost::system::error_code, std::size_t )> callback,
-                            bool to_sync_queue ) {
+                            uint32_t net_message_which ) {
```

Comment:
I feel this `uint32_t net_message_which` could be replaced with `msg_type_t msg_type`, where `msg_type_t` is defined as:
```
   enum class msg_type_t {
      signed_block       = fc::get_index<net_message, signed_block>(),
      packed_transaction = fc::get_index<net_message, packed_transaction>(),
      vote_message       = fc::get_index<net_message, vote_message>()
   };
```

Reply:
https://github.com/AntelopeIO/spring/pull/1090/commits/69804513b746746062e789fe6aba5237a00f098e